README.md

```diff
@@ -70,8 +78,10 @@ vaadin {
 
 All configuration options follow. Note that you **RARELY** need to change anything of the below.
 
-* `productionMode = false`: Whether or not we are running in productionMode.
-  The `vaadinBuildFrontend` task will automatically switch this to true, there is no need for you to configure anything.
+* `productionMode = false`: Whether or not the plugin should run in productionMode. Defaults to false.
+  Responds to the `-Pvaadin.productionMode` property. You need to set this to `true` if you wish
+  to build a production-ready war/jar artifact. If this is false, the `vaadinBuildFrontend`
+  task is automatically skipped.
```

Comment:
This is sort of false as in principle you could build a development mode war that doesn't start the devServer by running build-frontend with flow-build-info containing the property  `enableDevServer=false` which makes it possible tohave a "development mode" war that can be executed on other servers.

Reply:
Thanks! In such case the `enableDevServer` configuration should be also exposed in VaadinFlowPluginExtension. But first, can you please explain what is the advantage of having a development mode WAR with webpack bundle? Such an information would be useful to have documented both in README and in the javadoc.

Reply:
The use case is basically being able to get all the development time log and features when testing the war for instance on an external server.

The original issue is probably:
https://github.com/vaadin/flow/issues/5876

Reply:
I can not find the `enableDevServer` configuration property in flow-maven-plugin sources, in the 2.1 branch. Is it something new?

Reply:
Maven plugin sets it in `BuildFrontendMojo.updateBuildFile`

Reply:
Right, the one from the `flow-build-info.json` file. Thanks, should now be fixed.